### DISCUSSION TITLE
Core team meeting update: May and June 2023

### DISCUSSION THREAD

**Discussion by malvikasharan** [the-turing-way/the-turing-way#3215] [Governance]
On June 9, the second Core team meeting of 2023 was held, after a community call with the Core team to share updates from their work. The following documents were shared prior to the Core team meeting:

- Summary of the community status - here is the public version without the sensitive information/documents: https://docs.google.com/document/d/11WK59efJXRlYSxLompg5wyltA4ZR_-Uyez3JX0Wr1lc/edit?usp=sharing
- Updates from the different working groups/areas of work: https://docs.google.com/presentation/d/1Nj5qjKUGAd8hAzTQ7f4_ab_wiSlWurIQduzp23Mj1Xg/edit?usp=sharing 
  - Introduction of the Working Group’s work in November 2022 Community Share Out: https://www.youtube.com/watch?v=Z59KNFfWkkg. 

Chaired by Kirstie, Anne and Alex, The Turing Way staff team invited questions from the Core Team members around which they are seeking clarity and would like to improve documentation/resources - which have been listed and summarised below under different categories of actions needed:

1. **Update style guide, template and consistency guide** -- community solicitation, supported by community managers and the core staff

- Who designs fun new things like a new landing page?
    - Design suggestions have happened on an ad-hoc basis, with more experimental ideas usually being acted on at Book Dash
- Who decides if an edit should be a new chapter or just an edit?
    - [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md) - Project members - could be more clear but it is there
- Who to consult if you want to revise or restructure a chapter?
    - [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md) - Project members - could be more clear but it is there, Update style guide, template and consistency guide
        - should also be part of the contribution guideline
- How long can sections/chapters etc be?
    - The [Consistency Chapter](https://the-turing-way.netlify.app/community-handbook/consistency/consistency-structure.html) and [Chapter Template Folder](https://github.com/alan-turing-institute/the-turing-way/tree/main/book/templates/chapter-template) contains lots of relevant info but not exactly the length of the chapters/sections
- How to represent whose opinion is reflected in a subchapter/chapter (not even resolving, just showing)?
    - The original chapters are proposed on issues, discussed with other members, and the editing happens. The book is written by people, and reviewed by the community. Enhancement Proposals are suggested through issues as well. Not sure what this question means. Someone in the HackMD commented: [democratic mediums](https://medlabboulder.gitlab.io/democraticmediums/). It should definitely be a simpler process than JEPs - Django -- clarify what they mean by whose opinion beyond the people writing and reviewing.
        - Update style guide, template and consistency guide -- should also be part of contribution guideline. Need a high-level decision-making committee to oversee particularly thorny examples. Additionally, we can articulate guiding principles - human rights. Consider the tension between a transparency statement (the viewpoint of the author) vs not having individual authorship on chapters.

2. **Update contribution guideline** -- community solicitation, supported by community managers and the core staff

- Update from the previous section
- transparency vs not being able to find things
    - The information should be easy to find to avoid undocumented structures of power. *Should this be in the onboarding process?* KW note: that this can never be addressed! Relationships are powerful and should always be powerful!
        - Also part of governance - The team is documenting the three levels of decision-making. More clarity around who and where decisions get made.
- Add the review process
    - A chapter exists but doesn't define The Turing Way process
    - Project members - review, assign and approve PR. The name of the section could be changed for better understanding? [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md)
        - Ideally, community managers and infrastructure leads should be doing this. But recently we also discussed that a list of potential reviewers can be added - so people can identify who they can ask for help.
- Who do I _nominate_ to review a PR
    - Project members - review, assign and approve PR. The name of the section could be changed for better understading. [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md)
- When a piece of work is "done" enough to submit a PR +1
    - This is not explicitly mentioned. The contributor can decide it and the reviewer will approve or add comments. Contributing guidelines say open early and often. Make it explicit in Contribution Guideline. 
        - Capture the workflow in the community handbook.
- Who should fix a broken link if it is urgent (eg, the home page doesn't load)?
    - There's a [get in touch](https://github.com/alan-turing-institute/the-turing-way#get-in-touch) section.. We could specify urgent issues.
    - Ideally, community managers and infrastructure leads should be doing this. But recently we also discussed that a list of potential reviewers can be added - so people can identify who they can ask for help.
- Who should fix a broken link if it is not urgent?
    - Process for the good first issue
- Who resolves differences of opinion between the author of a PR and the reviewer?
    - This sounds like something that officially should be in the remit of the CM as a mediator but may be very context dependent - Make it explicit in Contribution Guideline. Needs a high-level committee to make a final decision on thorny issues. Links to other points around whose opinion is represented above.
- Who merges a PR?
    - Project members - review, assign and approve PR. The name of the section could be changed for better understading. [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md)
    - Make it clear that to merge, the author needs to have written permission on GitHub.

3. **Need an out-of-scope chapter in the Community handbook**  -- community solicitation

- What is and is **not* appropriate content for the book? Who decides if the content is out of scope for TTW?
    - Project members review and approve commits, issues and PR. They can identify NOT appropriate content for the book or raise it for discussion - Make it clear. [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md). The community monitors and adapts the content of the book and the project members review, guide and approve changes. Note that there are multiple levels at which to approach this question.
- How do we avoid becoming overly broad? What content is appropriate for TTW
- TTW will never be finished - but how do we define when things are out of scope and should not be added to the book? is there a point where we have all the themes or guides that are necessary and we keep building on those?

4. **Write a 'Tools we use' chapter** -- assign to the infrastructure leads

- https://github.com/alan-turing-institute/the-turing-way/issues/3039
- How do we utilise GitHub?
  - The translation team has their own organisation and the request for a separate org
- Who decides things in TTW? Who do we consult for big changes e.g. Jupyter -- also a huge part of governance work

5. **External collaboration chapter** -- assign to the Core Staff

- What an external collaboration with TTW could look like. Very interested to hear how to guide people here, I guess we could use The Practitioners Hub as an example(?) Lots of opportunities here and multifaceted.
    - Notes exist internally to the Turing but need auditing to make them more visible to external community members or organisations. Notes are technically available in the reporting folder but need to be improved. 

6. **Update CoC document** -- assign to the Community Manager
- How are conflicts between members resolved?
  - CoC applies equally to the project members
  - If the report is about a project team member, we have listed an ombudsperson to help with handling the report
  - Do we need more members? Like @Arielle-Bennett @gedankenstuecke who can join the committee?
- Who writes the CoC?
    - We have used open source CoC to build our CoC -- not written from scratch but uses existing/improved version that works for our community and with the resources we have. Details are at the end of the section.
    -  We can set a [CoC Committee](https://github.com/alan-turing-institute/the-turing-way/blob/main/CODE_OF_CONDUCT.md), but that is a resource-intensive work that we will return to in 2024 after the governance work have been addressed. 
        - **Action for later**: CoCc should be set - but also need CoC training budget
- Who makes decisions on potential breaches of COC? Who enforces the CoC?
    - We have a committee, and info is listed 
      - Make sure the team members' info is clearly stated
      - Remove Ben Murton who is no longer the ombudsperson

7. **Update Collaboration Cafe and Coworking chapters** -- assign to the Community Manager

- Who can bring a topic to a collaboration cafe? 
    - [Collab Cafe Topic proposals](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html)
- Who can bring a guest to a collaboration cafe?
    - It's a community space open for anyone - so bringing a guest is just literally having a new member join. 
        - provide a bring a-guest checklist
- Who joins coworking on Monday?
    - Update the chapter to state that currently it is used for the Core team

8. **Book Dash Chapter update** - assign to the Book Dash committee

- Who comes to the book dash?
    - [ Who should apply to join the Book Dash](https://the-turing-way.netlify.app/community-handbook/bookdash.html)
        - The exceptions from the last round did not really work well - we will not make more exceptions.
        - The TPS members should be supported by their line manager in scoping their application - if not written well, will not be invited
            - Involve the committee members in recruiting and onboarding new members
- Who sets the timelines for book dash?
    - The TTW team has done it so far
        - Will get the committee established first and then help them to set the deadline and documents
- Who designs the structure of the book dash?
    - A structure already exists. Day-to-day operations are set, but there is a day lead to ensure that chairs are supported, The Book Dash Committee also discuss and finalises the social and contribution format - we have been reusing previous formats that are described in the book.
- What projects get accepted to Book dash? What is the overall objective of the Book Dash?
    - Book Dash committee to review and update the rubrics
- Who approves images (Scriberia drafts)?
    - Participants who contribute to the development of images are asked for any improvement they want
    - Update Book Dash chapter: https://the-turing-way.netlify.app/community-handbook/bookdash/bookdash-illustrator.html

9. **Update presenting about The Turing Way**  -- assign to the trainers and mentors working group, supported by the core team

- Chapter link: https://the-turing-way.netlify.app/community-handbook/presenting.html
- Who gives talks about the Turing way
    - merge info from the https://hackmd.io/@turingway/demo-intro and https://the-turing-way.start.page/
- Who applies & who is given the opportunity to talk at conferences
    - So far CM applies for most and involves other members by asking in the Slack
    - Provide what other pathways exist
- Who represents TTW at conferences/events in an official capacity
    - Anyone should be able to - we have a promotion pack
- retire the promotion pack (archive on GitHub)
- What events/training etc we run
    - We do have most of these written in the Community Handbook for community-run events, for submission to other organisations, these are generally either community-proposed or Core Staff led
- What talks and event opportunities are available
    - We have an Event document that has been publicised - link to the chapter, update the Slack guide - add Slack guide in the Community handbook
- How to get funding to go to a conference
    - We don't have a policy - so far CM looks for resources and communicates with the project leads and the project management team
        - TPS programme management to Create a policy - add to the community handbook - Create a chapter on representing The Turing Way

10. **Add Slack guide in the community handbook**  -- assign to the Community Manager, involve community members

- How long is too long to leave a post unanswered in TTW Slack
    - We have a Slack Guide - not sure if we do need to go into such granular detail. Ideally, CM and other community members keep an eye on the posts.
- slack is a lot - important messages can be lost quickly (many people are in multiple workspaces)
    - Update the Slack guide - add Slack guide in the Community handbook

11. **Governance and funding-related questions** -- assign to the Core Team and TPS staff to take take the lead on

- Who decides to apply for funding for The Turing Way?
- Which events does TTW sponsor?
- How do I get further information on the funding (past, present, future) of the TTW?
- How funds are spent?
- Who can apply for funding on behalf of the project?
- most visible issue mentioned above is the relationship between the Turing Institute and the Turing Way (especially regarding funding allocation). how can we make this relationship more clear?
- important to tease apart the relationship between TPS & staff members
- the role of power dynamics & hierarchies, and teasing out these relationships
- up to what point does the Turing Way have a specific vision (that comes from Turing) and a community-led vision (coming)

**Comment by KirstieJane** [comment]
Thank you @malvikasharan - really keen to hear feedback!

**Comment by malvikasharan** [comment]
Thank you, Kirstie. 
I am also sharing a view-only document of our Google sheet where the questions from the core team were reviewed: https://docs.google.com/spreadsheets/d/1gjAtPCgk8Izh-xB1iK56wDOTe2ygHq5NG_-WFEPX-40/edit?usp=sharing

**Comment by malvikasharan** [comment]
We will use this thread for discussing and updating on Point 5 and Point 11.

For rest of the points, there are issues assigned to different people and Anne.
